jwst_lib/meta.yaml

```diff
@@ -1,33 +0,0 @@
-about:
```

Comment:
Wasn't supposed to be in the public channel. Any packages that were derived from this recipe were purged from the main channel a while ago.